### PR TITLE
Update teams-privacy-oce-overview.md

### PR DESCRIPTION
**Pull request by HadeelTantawy**
Modified the URL Preview Service section by adding the Bing Content Validation Service that URLs might be sent to as well

### CODE REVIEW

**Comment by HadeelTantawy** [comment]
Hello @MicrosoftHeidi, could you please review and sign off my change?

File: Teams/teams-privacy-oce-overview.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ GIPHY is a cloud-backed service that lets you use GIFs in your Teams chats. If y
...
+The URL Preview service automatically generates a preview snippet and attaches under the snippet the URL when a user sends a URL string. This service will make a request to the service URL as the user is typing the message. If the service URL doesn't have any schema.org data, it'll send a request to Bing search to get the data it needs to generate the preview snippet. Experiences that rely on Bing are licensed to you under the terms of the [Microsoft Services Agreement](https://www.microsoft.com/servicesagreement) and covered by the [privacy statement](https://privacy.microsoft.com/privacystatement). Any URLs you provide to the Microsoft Teams while using these services can be sent to Microsoft Bing, including Bing Content Validation Service for evaluation of harmful content. They are not linked to you by the Bing organization.
```
Comment (JohanFreelancer9) [inline Teams/teams-privacy-oce-overview.md:37]:
```suggestion
The URL Preview service automatically generates a preview snippet and attaches under the snippet the URL when a user sends a URL string. This service will make a request to the service URL as the user is typing the message. If the service URL doesn't have any schema.org data, it'll send a request to Bing search to get the data it needs to generate the preview snippet. Experiences that rely on Bing are licensed to you under the terms of the [Microsoft Services Agreement](https://www.microsoft.com/servicesagreement) and covered by the [privacy statement](https://privacy.microsoft.com/privacystatement). Any URLs you provide to Microsoft Teams while using these services can be sent to Microsoft Bing, including Bing Content Validation Service for evaluation of harmful content. They are not linked to you by the Bing organization.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @HadeelTantawy, not related, but could you please also commit the suggested change to fix the minor typo? Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by HadeelTantawy** [comment]
@MicrosoftHeidi @JohanFreelancer9 I don't see a merge option, most probably because I don't have write access to the repository. Can someone please merge the PR?

**Comment by JohanFreelancer9** [comment]
> @MicrosoftHeidi @JohanFreelancer9 I don't see a merge option, most probably because I don't have write access to the repository. Can someone please merge the PR?

@scanum, could you please assist? Thanks.

**Comment by scanum** [comment]
@MicrosoftHeidi 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**